cdap-ui/app/hydrator/controllers/detail-ctrl.js

```diff
@@ -89,8 +98,11 @@ angular.module(PKG.name + '.feature.hydrator')
       if (runsPoll) {
         runsPoll.unsubscribe();
       }
-      actionCreator.reset();
+      if (metricsObservable) {
+        metricsObservable.unsubscribe();
```

Comment:
does this actually stop the poll? I have suspicion that it won't

Reply:
Talked with @elfenheart, did a test and we found that the poll was being stopped when navigating away from the page.